test/acceptance_with_go_client/filters_tests/where_test.go

```diff
@@ -12,12 +12,13 @@
 package filters_tests
 
 import (
-	acceptance_with_go_client "acceptance_tests_with_client"
 	"context"
 	"fmt"
 	"testing"
 	"time"
 
+	acceptance_with_go_client "acceptance_tests_with_client"
+
```

Comment:
why did this import move? my formatter doesn't change this order

Reply:
It's correct go formatting

https://google.github.io/styleguide/go/decisions.html#import-grouping

Reply:
Which linter do you use (we agreed to use `golangci-lint` as a linter)? your import refactor makes my linter to mark the code in yellow saying that the file is not gofumted

Reply:
I am using golangci-lint as the linter

> jeremyprice@Jeremys-MacBook-Pro ~/g/weaviate (weaviate-core-refactor)> golangci-lint run                                                                                                                                   (base) 
jeremyprice@Jeremys-MacBook-Pro ~/g/weaviate (weaviate-core-refactor)> git diff                                                                                                                                            (base) 
jeremyprice@Jeremys-MacBook-Pro ~/g/weaviate (weaviate-core-refactor)>